src/components/Registration/RegistrationContainer.tsx

```diff
@@ -159,17 +162,17 @@ export const RegistrationContainer = withRouter(props => {
           <RegistrationStepOne
             onPecCheckboxChange={handlePecCheckboxChange}
             onScopeCheckboxChange={handleScopeCheckboxChange}
-            institutions={institutions}
-            onInstitutionSearch={handleIntitutionSearch}
-            onInstitutionSelected={handleInstitutionSelected}
-            selectedInstitution={selectedInstitution}
+            administrations={administrations}
+            onAdministrationSearch={handleIntitutionSearch}
```

Comment:
handleAdminstrationSearch ? (there's a typo anyway)

Reply:
fixed

Reply:
the typo prevented me to find the string to substitute